RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -1712,107 +1723,160 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
         else {
           continue;
         }
+
         unsigned int nInnerTriplets = tripletsOccupancy.nTriplets()[lowerModule1];
-        for (unsigned int innerTripletArrayIndex : cms::alpakatools::uniform_elements_y(acc, nInnerTriplets)) {
+
+        alpaka::syncBlockThreads(acc);
+
+        // Step 1: Make inner and outer triplet pairs
+        for (unsigned int innerTripletArrayIndex = threadIdY; innerTripletArrayIndex < nInnerTriplets;
+             innerTripletArrayIndex += blockSizeY) {
```

Comment:
this for loop should be identical to the previous approach: 
```c++
for (unsigned int innerTripletArrayIndex : cms::alpakatools::uniform_elements_y(acc, nInnerTriplets)) {
```

why the change ?